### DISCUSSION TITLE
Support unset

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#9568] [Ideas]
Hi can you please support syntax `...-unset`
Example: `top-6 lg:top-unset`  // 
I am using top-auto now, but it is not really right way.

**Comment by adamwathan** [comment]
I think `top-auto` here is fine personally since that's the default browser value you are reverting to, but you can also do `top-[unset]` or add `unset` as a value in your config file 👍🏻
